### PR TITLE
Issue Command

### LINKED ISSUE #70: Issue command
Similar to the PEP command (#67), but instead with GitHub issues. Could potentially show an embed with information regarding the issue.

### PR DESCRIPTION
**Pull request by MrDogeBro**
## Description

This adds the issue command. It is similar to the PEP command (#67) as well as the YouTube search command. If you do `+issue 1` or `+issue #1`, it gets the issue with the number 1 from this repo. It checks if the number is a valid issue number. If the query provided is not an number (or number preceded by a `#`), it searches the issues with the provided query. It uses the same menu to display and allow selection of these results that the YouTube search command does. When you select an issue, it displays the info for that issue. When using a text search query, if there is only one issue in the search results, it skips the menu and jumps straight to that issue as if you had specified a issue number.

Also, this command uses the GitHub API via a module called [PyGithub](https://pypi.org/project/PyGithub/). You will need to have an API token (personal access) for this to work. The token does not need any scopes as all it is doing is reading the data, not modifying/creating anything. Me and @parafoxia have alreadly discussed this some. If you need any help with getting the token and all configured/put into the bot, please DM me on discord. Thanks!

## Closes

 - Closes #70

### CODE REVIEW

File: carberretta/bot/cogs/meta.py [hunk 1]
```diff
@@ -20,11 +20,92 @@
 
 import discord
 from discord.ext import commands
+from github import Github, UnknownObjectException, Issue
```
Comment (parafoxia) [inline carberretta/bot/cogs/meta.py:23]: Sort these imports alphabetically.

```suggestion
from github import Github, Issue, UnknownObjectException
```

File: carberretta/bot/cogs/meta.py [hunk 2]
```diff
@@ -162,6 +244,41 @@ async def command_source(self, ctx: commands.Context, command: t.Optional[conver
...
 
+    @commands.command(name="issue")
+    async def command_issue(self, ctx: commands.Context, *, issue_query: str) -> None:
+        try:
+            issue_number = int(issue_query.lstrip("#"))
+
+            try:
+                issue = self.gh.get_repo("Carberra/Carberretta").get_issue(number=issue_number)
+            except UnknownObjectException:
+                await ctx.send("Invalid issue number")
```
Comment (parafoxia) [inline carberretta/bot/cogs/meta.py:255]:
```suggestion
                await ctx.send("Invalid issue number.")
```

File: pyproject.toml [hunk 3]
```diff
@@ -23,6 +23,8 @@ psutil = "^5.8.0"
 pygount = "^1.2.4"
 python-dotenv = "^0.13.0"
 toml = "^0.10.1"
+PyGithub = "^1.55"
```
Comment (parafoxia) [inline pyproject.toml:26]: Move this above kaomoji to make it alphabetical.

**Review by parafoxia — CHANGES_REQUESTED** [review]
Only minor stuff to change. Also, did you run the black beautifier on it? If not, run `black . -l119`. If multiple files change, *only* commit this one, and let me know about the others.

File: pyproject.toml (hunk 3 above)
Comment (parafoxia) [reply pyproject.toml:26]: Sorry above pygount, not kaomoji.

Comment (MrDogeBro) [reply pyproject.toml]: Done

**Review by parafoxia — APPROVED** [review]

**Pull request merged**